test/FisherPrice_Alpaka/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# This makes sure cmake knows where to find alpaka
+set(CMAKE_PREFIX_PATH "/install")
```

Comment:
This should be removed and instead the README updated with instructions on pointing to the Alpaca install.

Reply:
Is there a reason not to use the following setup? If alpaka is installed in the standard default area it will build it, otherwise it prints a message (if either the Alpaka is not in the default area or if the find_package command fails) instructing the user to check the README for the instructions about alpaka. Everything else will build ok in either scenario.

Cheers,

Mark

# This makes sure cmake knows where to find alpaka, if it was installed here
if (EXISTS "/install/include/alpaka/")
  set(CMAKE_PREFIX_PATH "/install")
else()
  message(STATUS "Could not find /install/include/alpaka, disabling build of FisherPrice_Alpaka. If you would like to build this please check the README for details of how to install Alpaka.")
endif()
# Then we must specify that we want to use alpaka
find_package(alpaka)

if(NOT alpaka_FOUND)
  message(STATUS "No alpaka install found, disabling build of FisherPrice_Alpaka. If you would like to build this please check the README for details of how to install Alpaka.")
  return()
else()  
  alpaka_add_executable(FisherPriceAlpaka alpaka_fisher_price.cu)
  target_link_libraries(FisherPriceAlpaka PUBLIC alpaka::alpaka)
endif()

Reply:
I'd rather we didn't introduce a hard coded path that's specific to one machine, even if only temporary. It's easy enough to set `CMAKE_PREFIX_PATH` local via command line, or as an environment variable.